### PR TITLE
Pizza crates cost 10k and need kitchen access to open

### PR DESCRIPTION
**Pull request by cacogen**
## About The Pull Request

The title and also adds the secure kitchen crate and fixes a bug with bought pizzas where their labels weren't showing up.

## Why It's Good For The Game

I was playing cook today and headed to cargo to order a food crate. There were bounty cubes piled up on the desk and nobody was doing their jobs. But that didn't matter because somebody had ordered pizza. I stole probably four of them and there was enough food for the station for the rest of the shift.

Pizza crates as they are remove the need for people with access to the pizzas to interact with the kitchen (which involves leaving one's department) and entirely negates the need for the cooks because ordering one is easier than cooking and not expensive enough to be cost prohibitive.

I usually play cargo and I've been in the position of ordering pizza for cargo when the kitchen has failed as well as leaving it at the kitchen when there's no food available. I don't usually play cook though and see the time and effort involved in making interesting things completely negated by a pizza order.

This is one way of dealing with the problem, by making it more expensive by cargo standards (though this depends on the round) and requiring access cargo doesn't have to open.

Another way of dealing with this is reducing the nutritional value of pizza, which I haven't looked at but may depending on the feedback this gets.

## Changelog
:cl: cacogen
balance: Pizza crates cost $10k and require kitchen access to open because they negated the need for the cook
fix: Bought pizzas have their labels properly show again
/:cl:

### CODE REVIEW

File: code/modules/food_and_drinks/pizzabox.dm [hunk 1]
```diff
@@ -289,24 +289,24 @@
...
-	. = ..()
 	pizza = new /obj/item/food/pizza/meat(src)
 	boxtag = "Meatlover's Supreme"
+	. = ..()
 
 /obj/item/pizzabox/pineapple/Initialize()
-	. = ..()
 	pizza = new /obj/item/food/pizza/pineapple(src)
 	boxtag = "Honolulu Chew"
+	. = ..()
```
Comment (Fikou) [inline code/modules/food_and_drinks/pizzabox.dm:309]:
```suggestion
	return ..()
```
do this for the others too

**Comment by ArcaneMusic** [comment]
Ihungry PR
I could probably get behind a nerf to their cost, but 50 crates of cargo might be a bit much.
That said, still a better direction than before if it's a real concern.


Now, one thing I DO know I disagree with is throwing department access on this on top of the price hike, since it also can't be bought by heads of staff for their department as a reward.

**Comment by Zytolg** [comment]
To paraphrase what Arcane said, this basically scuffs the idea of pizza parties as a reward. Locking pizzas on top of that, it's an overreaction to a minor problem. It's not fun having your job made obsolete by the actions of other station crew, but it means one of two things. 1) The Chef wasn't doing a good enough job so somebody decided they'd order a ton from cargo, or 2) Someone wanted a lot of pizza. I can name several ways I can make the Chef's job pointless on the station, and this is probably one of the more difficult options since you don't get a lot of pizza for what you spend on that crate currently. As it stands, this is only going to make it so that stations that host a low population have an increasingly difficult time feeding themselves. 

Not to say that everything in this PR is terribad. Fixing Pizzas is a good start, and a chef-unlockable box isn't necessarily a bad addition if it were to be used during the acquisition of some of the more exotic foods one might order from cargo (should it become an option). You probably don't want to lower pizza's nutrition (it's one of the easier foods for chef to create) either, but it would be a better alternative to what is currently being proposed. 

To me, this really reads as _"**x** ruined my game nerf **x**."_ and that's generally a bad basis for a change unless it's a recurring problem that is negatively impacting something consistently. But if I may suggest a course correction in thought, perhaps the issues at hand more lay with Cargo being able to buy an infinite amount of anything at once if given the means. Simply this is to say that putting a replenishable softcap on how much of something that might be ordered at a time is a better approach to limiting cargos ability to replicate this in the future, while maintaining the balance of pizza everywhere else.

**Comment by Tlaltecuhtli** [comment]
he died to a pizza crate

**Comment by cacogen** [comment]
@Zytolg
I don't see how this affects the pizza party thing and I don't consider it to be anywhere near as important as the cook being rendered obsolete by ordering a couple of crates. Given how simple and effective ordering pizza is vs. visiting the kitchen there's no reason not to do it at the moment other than courtesy to the cooks or somehow not having 2k as cargo.

I thought about it and the alternative to this would be to change the pizza in cargo to not be the same as the cook makes and less nutritional, rather than nerfing all pizzas and affecting the cook.

I doubt this would have any meaningful affect on people eating on lowpop. Less mouths to feed, pizza crates go further. It's not hard to prepare food yourself if there's no cook either.

I was aware of the inevitable accusations of this being an ided PR when I made it. I chose to be honest about what created the need for this because honesty usually leads to better outcomes if you're dealing with people who are looking for the truth of a matter instead of using it against you to win an argument.

I've always found the kneejerk "ided" response people have learned here to be braindead. It's definitely true some of the time but isn't necessarily the case. If someone sees something they think needs fixing and immediately goes ahead and tries to do it that doesn't mean the thing isn't broken.

@ArcaneMusic 
10k isn't much for the nutritional output five pizzas provide in the time it takes to get them compared to what the cook can output in the same time.

In the event a head of staff throws a pizza party they can probably find someone with kitchen access to open the crate.

**Comment by MrDoomBringer** [comment]
ihungry tag when

**Comment by Tlaltecuhtli** [comment]
if you want chef to not be joke job nerf the donuts that  is 90% of crew diet instead of once in a week crate that costs like 10 laser guns

**Pull request closed without merging**